src/state/store.ts

```diff
@@ -47,13 +47,22 @@ export const clearState = () => {
 
 /**
  * These actions make the redux devtools crash. Blacklist them from the developer tools.
+ * remove the blacklist for local debugging, but don't commit it
```

Comment:
market data is getting obese and bogging down dev tools - this makes it snappy for most devving, and can be disabled by commenting bits out as required